host-configuration/docs/1-verify-build-host.md

```diff
@@ -0,0 +1,29 @@
+# 1. Verify Your Build Host
+
+In order to build a Microservice binding natively, you need to be using a
+supported Linux distribution.
+In general, a recent version of Debian, Ubuntu, OpenSUSE, and Fedora works.
+Following is a specific list of supported distributions:
+
+* [Debian](https://www.debian.org/releases/) 9.0
+* [Ubuntu](https://wiki.ubuntu.com/Releases) 16.04, 16.10, 17.10, and 18.04
+* [OpenSUSE](https://en.opensuse.org/openSUSE:Roadmap) Leap 15.0 and Tumbleweed
+* [Fedora](https://fedoraproject.org/wiki/Releases) 27, 28, 29, and Rawhide.
+
+Exporting the `DISTRO` environment variable defines the distribution.
+Following are examples:
+
+**WRITER NOTE:** Email is out to the group asking about these string values.
+How does a developer know what value to use?
+What are the values for supported OpenSUSE and Fedora distributions?
```

Comment:
I dont understand that writer note

Reply:
The note was to remind me that I don't really know how a user understands the exact string to use based on the distro. For example, it is not intuitive that if I have Ubuntu 16.04 as my distro that I use "xUbuntu_16.04" as the string in the export statement.  What if my distro was Fedora 28?  Is my string "xFedora_28"?  Or, "Fedora_28".  Are there rules for making up the string?